cmd/vela-worker/flags.go

```diff
@@ -26,6 +26,13 @@ func flags() []cli.Flag {
 			Usage:   "set hostname for the worker",
 		},
 
+		&cli.DurationFlag{
+			EnvVars: []string{"VELA_WORKER_CHECK_IN", "VELA_CHECK_IN", "CHECK_IN"},
```

Comment:
For consistency with the other env vars, I think we want this to be `WORKER_CHECK_IN` 👍 

```suggestion
			EnvVars: []string{"WORKER_CHECK_IN", "VELA_CHECK_IN", "CHECK_IN"},
```